python_katas/kata_1/test.py

```diff
@@ -38,29 +43,66 @@ class TestVerbing(unittest.TestCase):
     1 Katas
     """
 
-    def test_sample(self):
-        # your code here
-        pass
+    def test_two_char(self):
+        test_word = 'mu'
+        self.assertEqual(questions.verbing(test_word), "mu")
+
+    def test_gaming(self):
+        test_word = 'gaming'
+        self.assertEqual(questions.verbing(test_word), "gamly")
+
+    def test_ing(self):
+        test_word = 'ing'
+        self.assertEqual(questions.verbing(test_word), "ly")
+
+    def test_in(self):
+        test_word = 'cartin'
+        self.assertEqual(questions.verbing(test_word), "cartining")
 
 
 class TestWordsConcatenation(unittest.TestCase):
     """
     1 Katas
     """
 
-    def test_sample(self):
-        # your code here
-        pass
+    def daft_funk(self):
+        lyrics=['one', 'more', 'time']
+        self.assertEqual(questions.words_concatenation(lyrics), 'one more time')
+
+    def bob_marley(self):
+        lyrics=["don't", 'worry', 'about', 'a', 'thing,', 'cause', 'every', 'little', 'thing', 'is', 'gonna', 'be', 'allright']
+        self.assertEqual(questions.words_concatenation(lyrics), "don't worry about a thing, cause every little thing is gonna be alright")
+
+    def dolly_parton(self):
+        lyrics=["working", "9", "to", "5"]
+        self.assertEqual(questions.words_concatenation(lyrics), 'working 9 to 5')
+
+    def queen(self):
+        lyrics=['we', 'are', 'the', 'champions', 'my', 'friend']
+        self.assertEqual(questions.words_concatenation(lyrics), 'we are the champions my friend')
+
 
 
 class TestReverseWordsConcatenation(unittest.TestCase):
     """
     1 Katas
     """
 
-    def test_sample(self):
-        # your code here
-        pass
+    def reverse_original_example(self):
```

Comment:
each function in the test case class must start with `test_`.
In your case: `test_reverse_original_example`